bots/zephyr_mirror_backend.py

```diff
@@ -596,6 +619,7 @@ def zcrypt_encrypt_content(zephyr_class, instance, content):
     return encrypted
 
 def forward_to_zephyr(message):
+    # type: (Any) -> Any
```

Comment:
Return type should be Any i guess in this case :
bots/zephyr_mirror_backend.py:723: error: Return value expected
bots/zephyr_mirror_backend.py:724: error: "send_error_zulip" does not return a value